.github/workflows/pypi-publish.yml

```diff
@@ -25,8 +25,8 @@ jobs:
           pipenv run pip3 install build setuptools wheel twine
       - name: Build and publish
         env:
-          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
-          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+          TWINE_USERNAME: __token__
```

Comment:
Should I replace this by something else? 😅 
Sorry for the noobie question

Reply:
The last time I did a publish with twine it was still using user names and passwords so I haven't tried this, but I think `__token__` is correct here. From what I can find I think it tells twine that the password will be the token.